app/views/tran_details/_form.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= form_with(model: [ @tran_group, @tran_detail], url: tran_group_tran_details_path(@tran_group, @tran_detail), local: true) do |form| %>  <% if tran_detail.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(tran_detail.errors.count, "error") %> prohibited this tran_detail from being saved:</h2>
+      <ul>
+        <% tran_detail.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+  <div>
+    <%= form.label :itemDetails, style: "display: block" %>
+    <%= form.text_field :itemDetails %>
+  </div>
+  <div>
+    <%= form.label :amount, style: "display: block" %>
+    <%= form.text_field :amount %>
+  </div>
+  <div>
+    <%= form.label :tran_group_id, style: "display: block" %>
+    <%= form.collection_select :tran_group_id, @available_groups, :id, :groupName, prompt: "Select a Group" %>
+  </div>
+  <br />
+  <div>
+    <%= form.submit 'Add Transaction', class: "btn btn-outline-success"%>
```

Comment:
- [ ] When I click on this button when the form fields are empty, I got the error on the screenshot below. Kindly validate your inputs correctly. Error messages should display instead of the browser error :+1: 
![Screenshot from 2023-09-22 11-10-19](https://github.com/Nahid1911/budgetApp-RoR/assets/63087491/2311f41a-5bc5-4f3f-9d17-4d33249857ad)

Reply:
resolved